dwave/system/samplers/clique.py

```diff
@@ -184,6 +185,18 @@ def __init__(self, *,
         self.child = DWaveSampler(
             failover=failover, retry_interval=retry_interval, **config)
 
+    def close(self):
+        """Close the child sampler in order to release system resources like threads.
```

Comment:
```suggestion
        """Close the child sampler to release system resources such as threads.
```

Because you'll appreciate [this](https://www.grammarly.com/blog/grammar/like-vs-such-as/) little subtlety/nitpick.  😄 

Elsewhere too.